### ISSUE TITLE
Reading age

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#258]
We say that "we aim for a reading age of 9 to 11 years old. But we recognise that, with some medical information, it's not easy to achieve this. In this case, try to make sure that an 11 to 14-year old will understand."

"Reading age" is a problematic concept. Rather than talking about "reading age", should we refer to  functional literacy and literacy skills?

1 in 6 adults in England have very poor literacy skills. (Skills for Life Survey, 2012 and OECD Survey of Adult Skills, 2015) 
https://literacytrust.org.uk/parents-and-families/adult-literacy/
https://literacytrust.org.uk/parents-and-families/adult-literacy/what-do-adult-literacy-levels-mean/

**Comment by sarawilcox** [comment]
I'll try and write up what we've learnt about "reading age" so that we can bring it to Style Council.

**Comment by sarawilcox** [comment]
See also W3C WCAG's [gap analysis of the state of accessibility for People with learning disabilities and cognitive disabilities](https://w3c.github.io/wcag/coga/gap-analysis.html#introduction). Looks at the needs of a range of users, including people with dyslexia, aphasia, dementia and Down's syndrome.

**Comment by sarawilcox** [comment]
### January Style Council meeting

We say this in the style guide: "We aim for a reading age of 9 to 11 years old. But we recognise that, with some medical information, it's not easy to achieve this. In this case, try to make sure that an 11 to 14-year old will understand."  

There are issues with the term “reading age” and giving particular age ranges. (More detail in the presentation slides.) 

At the meeting, we were not able to agree any changes to the content style guide. (The proposal was to move away from referring to a specific reading age.) Instead people commented that: 

- it can be helpful to mention reading age when talking to stakeholders  
- it may – or may not - be helpful to think about a particular age when writing a specific piece of content – for those who feel they know what a child of a given age can read 
- we should explain in more detail what very poor literacy skills mean, with examples, e.g. from the Literacy Trust and add that low literacy can change users’ reading behaviour.  
- we should be clear that reading age doesn’t reflect intelligence 

We agreed that Sara will redraft something for the next Style Council meeting, if possible, which: 

- mentions 9 to 11 year-old language skills, with examples 
- takes into account the comments above 
- mentions the other groups of users in the slides (e.g. non-English speakers)

**Comment by sarawilcox** [comment]
We're looking at doing some research in the content profession to get a clearer understanding of content designer user needs for information about reading age and low literacy levels.

**Comment by sarawilcox** [comment]
I'm putting this issue back into In Progress until we can carry out some user research with content designers.

**Comment by sarawilcox** [comment]
Nielsen Norman Group studies show that users with lower literacy read things differently:

- they plough through text, word by word, and have problems scanning it
- they struggle with words of many syllables
- they miss page elements because they have a narrower field of view
- they may skip large amounts of information if it's too complicated

Also, from the Literacy Trust, people with very poor literacy skills "can understand short straightforward texts on familiar topics accurately and independently, and obtain information from everyday sources, but reading information from unfamiliar sources, or on unfamiliar topics, could cause problems."

"A 2011 government survey of adult literacy skills found that 14.9% (or 1 in 7) of adults in England have literacy levels at or below Entry Level 3, which is equivalent to the literacy skills expected of a nine to 11-year-old", which means that they "may not be able to understand labels on pre-packaged food or understand household bills".

The 1 in 6 figure is from the OECD basic skills survey 2015.

**Comment by sarawilcox** [comment]
We have received feedback suggesting that we remove the reference to writing for 9 to 11 year olds. The feedback:
- asks how we arrived at the advice
- says that ONS has now withdrawn its statement that the "average reading age" of UK adults is 9 years old
- questions the concept of "reading age" 
- suggests that we should aim for level 1 in terms of functional literacy 
(In fact, it looks like GOV.UK and Content Design London no longer refer to writing for 9 to 11 year olds.)

**Comment by sarawilcox** [comment]
ADD STYLE COUNCIL NOTES HERE WHEN APPROVED.

In the meantime, we have had a conversation with the Patient Information Forum whose members, like NHS.UK content designers, find it helpful to have something to aim for, with caveats.

Next steps: follow up with user research.

**Comment by sarawilcox** [comment]
Recent comments on GOV.UK Slack: 
There's evidence that 15% of adults have literacy skills below level 3, which is the level generally expected of a 9 to 11 year old.
But reading ages and other types of readability formulae have limitations as a way of evaluating website content: https://www.effortmark.co.uk/readability-formulas-seven-reasons-to-avoid-them-and-what-to-do-instead/

Plus mention of a doc for people with learning disabilities that is coming out as post-graduate level in a reading algorithm.

**Comment by sarawilcox** [comment]
### From the Patient Information Forum
A survey of PIF members has revealed strong support for a target reading age of 9-11 years old for health information.

The survey, developed with our Health Literacy Expert Panel, took place in late January and early February. It was carried out in response to a non-member challenge to our recommendation.

Of the 93 information leads who responded, 95% support our policy to aim for a reading age of 9-11.

[More info on PIF's website](https://pifonline.org.uk/news/target-reading-age-health-information/)

**Comment by sarawilcox** [comment]
[Local action on health inequalities: Improving health literacy to reduce health inequalities](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/460710/4b_Health_Literacy-Briefing.pdf)

In the European Health Literacy Survey (EU-HLS), which includes measures of all three levels of health literacy – functional, interactive and critical health literacy nearly every second respondent showed limited health literacy.

A study on functional health literacy levels across England reports that 42% of working-age adults in England are unable to understand and make use of everyday health information, rising to 61% when numeracy skills are also required for comprehension . Furthermore, 43% of working-age adults will struggle to understand instructions to calculate a childhood paracetamol dose.

**Comment by sarawilcox** [comment]
We've received further information from the Patient Information Forum and colleagues at Health Literacy UK about their survey findings, with a request/joint recommendation that we retain our info about reading age in the style guide.

**Key findings from a survey of information leads in PIF member organisations**

- 95% supported our policy to aim for a reading age of 9-11
- 59% of organisations have a formal policy on reading age, 43% aimed for 9-11, 16% aimed for 12-14). A further 24% had other measures to support the use of plain English.

When we asked individuals responding about their own practice, a higher proportion were aiming for a target reading age when developing content.
- 68% aim for a reading age (48% 9-11 and 19% 12-14)
- 19% used style guides or other measures to ensure content is written in plain English.

Making information as accessible as possible was given as the main benefit of setting a target reading age (83%). Providing consistency for the team (47%), providing guidance to external writers (22%) and guidance for medical advisors (15%) were identified as other benefits.

Readability tools were used by 56% of respondents. The Hemmingway App is the most popular tool, used by 38%, followed by Microsoft Word’s readability checker (30%). Grammerly was used by 12% and SMOG 10%.

Further details in the attached letter.
[PIF reading age survey letter.pdf](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/files/8253096/PIF.reading.age.survey.letter.pdf)

I'll report this to next Style Council with a recommendation that we keep our existing guidance but consider providing further guidance about what writing for this reading age means.

**Comment by sarawilcox** [comment]
At April Style Council meeting, we agreed to follow PIF's research findings and keep references to reading age in the style guide. We may want to come back at some stage and add more guidance to explain what we mean by these specific reading ages. 

I don't think that we'll be able to do this in the short term, so I'm going to put this issue back into the To Do column for the time being.

**Comment by cjforms** [comment]
I'm really disappointed that the outdated and unhelpful concept of "reading age" was retained. 
Only came across this thread today - sorry about the tardy response.
